src/posts/2018-08-29-good-frontend-developer.md

```diff
@@ -86,7 +86,7 @@ Design is important.
 
 When I talk about design, I mean both visual design and interaction design.
 
-**We're the best person to work on design because we know how the web works**. We know what can be done and what cannot be done. We also know how people use the web. And we care enough to keep ourselves updated.
+**We're the best people to work on design because we know how the web works**. We know what can be done and what cannot be done. We also know how people use the web. And we care enough to keep ourselves updated.
```

Comment:
We is singular here, so person is still the right word imo. Can you change this back please? Thanks.

Reply:
I changed it Zell. Stay awsome :)